apps/fyle/helpers.py

```
@@ -322,84 +322,106 @@ def sync_dimensions(workspace_id, is_export: bool = False) -> None:
+                filter1 = {
+                    f'custom_properties__{expense_filter.condition}__in': expense_filter.values
+                }
+                # Invert the filter using the ~Q operator and assign it to the constructed expense filter
+                constructed_expense_filter = ~Q(**filter1)
+            else:
+                # If the custom field is of type NUMBER, convert the values to integers
+                if expense_filter.custom_field_type == 'NUMBER':
+                    expense_filter.values = [int(value) for value in expense_filter.values]
+                # If the expense filter is a custom field and the operator is yes or no(checkbox)
+                if expense_filter.custom_field_type == 'BOOLEAN':
+                    expense_filter.values[0] = True if expense_filter.values[0] == 'true' else False
```

Ruff (0.8.2)

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)